### PR TITLE
Nodes: export viewport and minor fixes

### PR DESCRIPTION
**Pull request by aardgoose**
As title.

### CODE REVIEW

File: examples/jsm/nodes/materials/Line2NodeMaterial.js
Comment (LeviPesin) [inline examples/jsm/nodes/materials/Line2NodeMaterial.js:1]: I think this file was added by mistake?

**Comment by LeviPesin** [comment]
Can you please also add MeshLambertNodeMaterial, LineDashedNodeMaterial, BumpMapNode, HashNode, and Schlick_to_F0? They were added in previous PRs but weren't, I think, added to the exports.

File: examples/jsm/nodes/materials/Line2NodeMaterial.js
Comment (aardgoose) [reply examples/jsm/nodes/materials/Line2NodeMaterial.js:1]: oops. removed

File: examples/jsm/nodes/accessors/LineMaterialNode.js [hunk 1]
```diff
@@ -4,12 +4,6 @@ import { nodeImmutable } from '../shadernode/ShaderNode.js';
 
 class LineMaterialNode extends MaterialNode {
 
-	constructor( scope ) {
-
-		super( scope );
-
-	}
-
 	construct( builder ) {
```
Comment (mrdoob) [inline examples/jsm/nodes/accessors/LineMaterialNode.js:7]: Hmm... Would it make sense to rename `construct()` to `build()`?

Comment (LeviPesin) [reply examples/jsm/nodes/accessors/LineMaterialNode.js:7]: I think `build` is already used for the last step -- turning the node into a shader code fragment. The `construct` name for the node constructing step seems nice 👍

Comment (mrdoob) [reply examples/jsm/nodes/accessors/LineMaterialNode.js:7]: Ah, maybe `setup()` then? Or `configure()`?
Just trying to differentiate it from `constructor()`...

Comment (sunag) [reply examples/jsm/nodes/accessors/LineMaterialNode.js:7]: `setup()` is interesting.

**Pull request merged**